src/Microsoft.AspNet.Mvc.Core/FormatterMappings.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Net.Http.Headers;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using Microsoft.AspNet.Mvc.Core;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// Used to specify mapping between the Url Format and corresponding <see cref="MediaTypeHeaderValue"/>.
+    /// </summary>
+    public class FormatterMappings
```

Comment:
`SetMediaTypeMappingForFormat` vs `GetMediaTypeForFormat` names can still be more symmetrical